api/src/opentrons/protocol_runner/protocol_runner.py

```diff
@@ -27,43 +31,76 @@
 )
 
 
-class ProtocolRunResult(NamedTuple):
+class RunnerRunResult(NamedTuple):
     """Result data from a run, pulled from the ProtocolEngine."""
 
     commands: List[Command]
     state_summary: StateSummary
 
 
-# TODO(mc, 2022-01-11): this class has become bloated. Split into an abstract
-# interfaces and several concrete implementations per protocol type
-class ProtocolRunner:
+class AbstractRunner(ABC):
```

Comment:
```suggestion
class AbstractProtocolRunner(ABC):
```
I think this will make naming related classes/ methods a lot easier.